daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/StablePackage.scala

```diff
@@ -198,14 +198,7 @@ private[daml] object StablePackage {
           "GHC.Tuple",
           "6839a6d3d430c569b2425e9391717b44ca324b88ba621d597778811b2d05031d",
           "daml-prim",
-        ) {
-      object Check
-          extends StablePackage(
-            "GHC.Tuple.Check",
-            "1b04d27b93f5c72a4cd7716feacae7b7aef27235869daaabd5fe07edb14d6002",
-            "daml-prim",
-          )
-    }
```

Comment:
By the way I saw that this package and  `DA.Internal.Interface.AnyView.Types` was not added to the list `StablePackage.stablePackages`. Maybe you could add the latter in this PR.

Reply:
since this PR is about _removing_ `GHC.Tuple.Check` as a stable package, I've added ` DA.Internal.Interface.AnyView.Types` in a separate PR, https://github.com/digital-asset/daml/pull/15559

Reply:
:+1: